### PR TITLE
add delay to Unconsciousness Captivity

### PR DESCRIPTION
**Pull request by jokoho48**
add delay to Unconsciousness Captivity(posible fix for #3121)

### CODE REVIEW

File: addons/medical/functions/fnc_setUnconscious.sqf [hunk 1]
```diff
@@ -94,8 +94,16 @@ _unit setUnitPos "DOWN";
 if (GVAR(moveUnitsFromGroupOnUnconscious)) then {
     [_unit, true, "ACE_isUnconscious", side group _unit] call EFUNC(common,switchToGroupSide);
 };
+// Delay Unconscious so the AI dont instant stop shooting on the unit #3121
+if (GVAR(delayUnconCaptive) == 0) then {
+    [_unit, QGVAR(unconscious), true] call EFUNC(common,setCaptivityStatus);
+} else {
+    [{
+        [_this select 0, QGVAR(unconscious), true] call EFUNC(common,setCaptivityStatus);
```
Comment (PabstMirror) [inline addons/medical/functions/fnc_setUnconscious.sqf]: Need to make sure they are still unconc at this point, if delayUnconCaptive > (wake up time)

Comment (jokoho48) [inline addons/medical/functions/fnc_setUnconscious.sqf]: oh god point i missed that

File: addons/medical/ACE_Settings.hpp [hunk 2]
```diff
@@ -223,4 +223,11 @@ class ACE_Settings {
         values[] = {CSTRING(useSelection), CSTRING(useRadial), "Disabled"};
         isClientSettable = 1;
     };
+
+    class GVAR(delayUnconCaptive) {
+        displayName = CSTRING(delayUnconCaptive);
+        typeName = "SCALAR";
+        value = 10;
+        values[] = {CSTRING(painEffect_Flash), CSTRING(painEffect_Chroma)};
```
Comment (esteldunedain) [inline addons/medical/ACE_Settings.hpp]: Not needed

Comment (jokoho48) [inline addons/medical/ACE_Settings.hpp]: oh right :)

**Comment by captainblaffer** [comment]
I don't know if this feature is good for gameplay.

For missions without revive module, which are quite hardcore players will be less likely to survive. Which imo is not fun. They die, medics don't have anything to do.

for missions with revive, it makes 0 sense for the AI to keep shooting the player for 10 seconds as they can't die anyway. It will only cause the player to get hurt more requiring more bandages(annoying). Or the AI will keep shooting the unc. player instead of the alive player standing next to them, which will make the AI look dumb.

**Comment by jokoho48** [comment]
@captainblaffer you can easy disable this if you replace the number with a 0 :D

**Comment by captainblaffer** [comment]
I understand, but that is quite inconvenient for my community at least. I hoped if the general opinion is in line with my thoughts, to set the default value at 0.

File: addons/medical/CfgVehicles.hpp [hunk 3]
```diff
@@ -138,6 +138,13 @@ class CfgVehicles {
                 typeName = "BOOL";
                 defaultValue = 1;
             };
+
+            class delayUnconCaptive {
+                displayName = CSTRING(MedicalSettings_delayUnconCaptive_DisplayName);
+                description = CSTRING(MedicalSettings_delayUnconCaptive_Description);
+                typeName = "NUMBER";
+                defaultValue = 10;
+            }
```
Comment (Glowbal) [inline addons/medical/CfgVehicles.hpp]: Missing `;`

**Comment by Glowbal** [comment]
I am ok with including this feature however:
- Please finish the translations. Also do mention that value of the setting is in seconds.
- I am debating what the default value for this setting should be. We will discuss in Slack @jokoho48

**Comment by esteldunedain** [comment]
> I am debating what the default value for this setting should be

My opinion is default off

**Comment by jokoho48** [comment]
or we make it 3 secs

**Comment by esteldunedain** [comment]
> or we make it 3 secs

Yeah, that could be about right too.

**Comment by SilentSpike** [comment]
I mentioned it on the focus feature issue, but I think having a setting for this is way too specific, if the default value is set right to begin with then a setting isn't meaningful to the average user.

**Comment by commy2** [comment]
^ No setting needed for this

**Comment by Glowbal** [comment]
Yes, I am with you on that @SilentSpike. I think using 3 seconds is plenty. We can adjust based upon user feedback.

**Comment by mickeymen** [comment]
I understand the lack of delay causes the ability of the AI to be merciful, to an unconscious opponent! 
#3121
Do I understand correctly? Thus, please add the short delay finally and solve the problem of the stupid behavior of the AI in the ACE3!  what are the obstacles to this?

**Pull request merged**